example/cfg/EdgeBattery.cfg

```diff
@@ -0,0 +1,98 @@
+# General configuration
+publishInvalid: true
+intervalDelay: 1000
+
+# Prognoser configuration
+model:Battery
+
+# Model Configuration
+Model.processNoise: 1e-5, 1e-5, 1e-5, 1e-5, 1e-5, 1e-5, 1e-5, 1e-5
+#This is a vector consisting of the variances of each process noise term. There exists a process noise term corresponding to every state in the model and is assumed to follow a Gaussian distribution with zero mean and variance specified in the configuration file.
+
+# Observer configuration
+observer:UKF
+#Unscented Kalman Filter(UKF): a Bayesian filtering algorithm for highly nonlinear dynamic systems.
+Observer.Q: 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10
+Observer.R: 1e-2, 0, 0, 1e-2
+
+# Predictor configuration
+predictor:MC
+#Monte Carlo(MC): sampling based prediction algorithm
+Predictor.numSamples: 10
+#Number of samples used by the UKF
+Predictor.horizon: 10000
+#Prediction Horizon depends on the type of battery packs and loading profiles. In case these change user needs to update the values so as to incorporate entire time frame of the projected usage profile
+
+# Setup for PlaybackScn2Constant
+Predictor.inputUncertainty: 284, 0.1
+
+# Edge Battery Config (These parameterrs are specific for a 5 cell battery pack. The parameters will change depending on the type of battery pack and number of cells in each of the pack)
+
+#An0- An9 - Empirical parameter for negative electrode in Redlich-Kister expansion.

```

Comment:
Since these are part of the same battery no need to added a comment to every parameter.